src/topp/QualityControl.cpp

```diff
@@ -78,6 +79,8 @@ class TOPPQualityControl : public TOPPBase
     //possible additions:
     //"mzData,mzXML,dta,dta2d,mgf,featureXML,consensusXML,idXML,pepXML,fid,mzid,trafoXML,fasta"
     registerFlag_("MS2_id_rate:force_no_fdr", "forces the metric to run if fdr was not made, accept all pep_ids as target hits");
+    registerFlag_("FragmentMassError:tolerance_unit_ppm", "unit for tolerance, default m/z");
```

Comment:
maybe make this a StringOption with two choices (see https://github.com/OpenMS/OpenMS/blob/develop/src/topp/XTandemAdapter.cpp#L150). Advantage is the user will see the unit explicitly in parameter settings.